hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/ReadableScheduleStore.java

```diff
@@ -56,13 +56,14 @@ public ReadableScheduleStore(@NonNull final States states) {
      */
     public Optional<ScheduleMetadata> get(final ScheduleID id) {
         final var schedule = schedulesById.get(id.getScheduleNum());
-        return schedule.map(
-                value ->
+        return schedule == null
```

Comment:
`return Optional.ofNullable(schedule).map(s -> new ScheduleMetadata(.....))` can be done instead of using `a ? c : d`